src/profile/Index.vue

```diff
@@ -1,89 +1,171 @@
+            <v-img :id="item.label" :src="item.image" max-width="40px" class="mx-auto"/>
+            <label :for="item.label">{{item.label}}</label>
+          </a>
+        </div>
       </v-col>
+    </v-row>
 
-      <v-col cols="12" sm="6" md="4">
-        <BackupCodeCard :meta="mfa.backup"/>
+    <v-row>
+      <v-col>
+        <v-footer>
+        <small>
+          <span v-for="(item, index) in googleItems" :key="index">
+            {{item.label + getSpaceCommaOrAnd(index, googleItems.length)}}
+          </span> are trademarks of Google LLC.
+        </small>
+        </v-footer>
       </v-col>
     </v-row>
   </BasePage>
 </template>
 
 <script>
 import ProfileProgress from './ProfileProgress'
-import PasswordCard from './PasswordCard'
-import PasswordRecoveryCard from './PasswordRecoveryCard'
-import TotpCard from './TotpCard'
-import SecurityKeyCard from './SecurityKeyCard'
-import BackupCodeCard from './BackupCodeCard'
-import DoNotDiscloseCard from './DoNotDiscloseCard'
-import Attribute from './Attribute'
-import { recoveryMethods, retrieve as retrieveMethods} from '@/global/recoveryMethods';
-import { mfa, retrieve as retrieveMfa } from '@/global/mfa';
+import { recoveryMethods, retrieve as retrieveMethods} from '@/global/recoveryMethods'
+import { mfa, retrieve as retrieveMfa } from '@/global/mfa'
+
 
 export default {
   components: {
-    ProfileProgress,
-    PasswordCard,
-    PasswordRecoveryCard,
-    TotpCard,
-    SecurityKeyCard,
-    BackupCodeCard,
-    DoNotDiscloseCard,
-    Attribute,
+    ProfileProgress
   },
   data: () => ({
     alternates: recoveryMethods.alternates,
     mfa,
   }),
   computed: {
-    hasUnverifiedEmails: vm => vm.alternates.some(m => ! m.verified),
+    org: (vm) => vm.$user.email.split('@')[1].split('.')[0],
+    // Todo update/add to locales
+    coreItems: (vm) => JSON.parse(vm.$root.idpConfig.coreItems || '[]').map(item => {
+      switch (item.title) {
+        case 'Example':
+          item.secondary = 'This is an example'
+          break
+        case 'Workday':
+          item.secondary = 'HR and organizational data'
+          break
+        case 'Gateway':
+          item.secondary = `${vm.org}'s internal wiki`
+          break
+        case 'Zoom':
+          item.secondary = 'Participate in virtual meetings'
+          break
+        case 'Help Desk':
+          item.secondary = 'Browse how-to articles and submit tickets'
+          break
+        case 'REAP':
+          item.secondary = 'Repo for Electronic Archiving and Publishing'
+          break
+        case 'Cover':
+          item.secondary = 'insurance'
+          break
+        default:
+          break
+      }
+      return item
+    }),
+    
+    googleItems(vm) {
+      return [
+        {
+          image: 'https://lh3.googleusercontent.com/0rpHlrX8IG77awQMuUZpQ0zGWT7HRYtpncsuRnFo6V3c8Lh2hPjXnEuhDDd-OsLz1vua4ld2rlUYFAaBYk-rZCODmi2eJlwUEVsZgg',
```

Comment:
Is this an official URL for the Gmail logo? Since it's on "googleusercontent.com", it seems a little suspicious, but I could be wrong. (Similarly with the other Google logos.)

Reply:
yes, I got them from [here](https://about.google/brand-resource-center/logos-list/).